srcpkgs/obs/template

```diff
@@ -1,14 +1,15 @@
 # Template file for 'obs'
 pkgname=obs
 version=31.0.2
-revision=2
+revision=3
```

Comment:
this should be 4, it was bumped to 3 in 6d7269f01b8f951ae30cf0f2dff9c151b2f703d1